code/modules/ventcrawl/ventcrawl.dm

```diff
@@ -3,6 +3,15 @@ var/global/list/ventcrawl_machinery = list(
 	/obj/machinery/atmospherics/unary/vent_pump
 	)
 
+/mob/living/carbon/human/AltClickOn(var/atom/A)
+	if(is_type_in_list(A,ventcrawl_machinery) && w_uniform && istype(w_uniform,/obj/item/clothing/under/contortionist))
+		var/obj/item/clothing/under/contortionist/C = w_uniform
+		if(C.check_clothing(src))
+			src.handle_ventcrawl(A,1)
```

Comment:
This will fail if it makes it to handle_ventcrawl, because the mob doesn't have the ventcrawl verb and won't pass the can_ventcrawl() check.